packages/plugin-ext/src/main/browser/webviews-main.ts

```diff
@@ -25,24 +25,41 @@ import { WebviewWidget } from './webview/webview';
 import { ThemeService } from '@theia/core/lib/browser/theming';
 import { ThemeRulesService } from './webview/theme-rules-service';
 import { DisposableCollection } from '@theia/core';
+import { ViewColumnService } from './view-column-service';
+
+import debounce = require('lodash.debounce');
+import { Cancelable } from 'lodash';
 
 export class WebviewsMainImpl implements WebviewsMain {
+    private readonly revivers = new Set<string>();
     private readonly proxy: WebviewsExt;
     protected readonly shell: ApplicationShell;
+    protected readonly viewColumnService: ViewColumnService;
     protected readonly keybindingRegistry: KeybindingRegistry;
     protected readonly themeService = ThemeService.get();
     protected readonly themeRulesService = ThemeRulesService.get();
+    protected readonly handler: (() => void) & Cancelable;
```

Comment:
would be nice to give a better name, i.e `updateViewOptions`.

Reply:
fixed

Reply:
fixed